nixos/modules/virtualisation/containers.nix

```diff
@@ -419,6 +423,7 @@ let
     {
       extraVeths = {};
       additionalCapabilities = [];
+      timeoutStartSec = null;
```

Comment:
I don't understand why i had to add this line, but without it the evaluation fails with this error:

`attribute 'timeoutStartSec' missing, at /home/davidak/code/nixpkgs/nixos/modules/virtualisation/containers.nix:258:23`

Reply:
Needed because of this:
https://github.com/NixOS/nixpkgs/blob/92f30b03a7145dc20b4b43d41685405a38ff1167/nixos/modules/virtualisation/containers.nix#L692-L693

So these values are used for this service template, where null gets translated to it not being set